Tests/CasePathsTests/CasePathsTests.swift

```diff
@@ -5,74 +5,73 @@ final class CasePathsTests: XCTestCase {
   func testEmbed() {
     enum Foo: Equatable { case bar(Int) }
 
-    XCTAssertEqual(.bar(42), (/Foo.bar).embed(42))
+    let fooBar = /Foo.bar
```

Comment:
Almost all tests now pre-assign case paths via inference in order to ensure no ambiguity.